src/main/java/uk/co/ramp/covid/simulation/Model.java

```diff
@@ -39,6 +38,7 @@
     private String networkOutputDir = null;
     
     private List<LockdownEvent> lockdownEvents = null;
+    private List<LockdownEventGenerator> lockdownGenerators = null;
```

Comment:
You could make the logic a little simpler (below) by initializing this to `ArrayList<>()` instead of `null`.

Reply:
Thanks.  I was imagining also removing the `if (lockdownGenerators != null)` (etc.).

Reply:
Thanks for pointing that out - I didn't realise they were still in there!